src/backend/marsha/core/models/video.py

```diff
@@ -147,15 +170,21 @@ def get_ready_clause():
         """Clause used in lti.utils.get_or_create_resource to filter the videos.
 
         Only show videos that have successfully gone through the upload process,
-        or live streams that are in the running state.
+        or live streams that are in the running state or videos that are in the scheduled mode.
 
         Returns
         -------
         models.Q
             A condition added to a QuerySet
         """
-        return models.Q(uploaded_on__isnull=False, live_state__isnull=True) | models.Q(
-            live_state__isnull=False
+        return (
```

Comment:
As explained in an other comment you must update the test in `lti_test.py` to test the wanted behaviour of the `get_or_create_resource`.

Reply:
I wonder after reading this file if I shouldn't add control on upload_state. I thought that as long as live_state changed, upload_state changed but maybe I should restrict it to pending? WDYT ?

Reply:
test